.github/workflows/trigger_azdo_ci.yml

```diff
@@ -1,7 +1,10 @@
 name: Trigger AzDO CIs
 
 on:
-  workflow_call:
+  workflow_run:
+    workflows: [CI]
+    types:
+      - completed
     secrets:
```

Comment:
I'm not entirely sure if it needs this `secrets` section anymore or if that was just needed when it was triggered by a `workflow_call`.

Reply:
I don't think that section was needed/useful. The secrets should be passed/available and the workflow already just references `secrets.AZDO_PIPELINE_TRIGGERS`.

Reply:
I don't think that section was needed/useful. The secrets should be passed/available and the workflow already just references `secrets.AZDO_PIPELINE_TRIGGERS`.